### PR TITLE
social recovery docs

### PR DESCRIPTION
**Pull request by YasseinBilal**

### CODE REVIEW

File: pages/module-sdk/modules/social-recovery-validator.mdx [hunk 1]
```diff
@@ -0,0 +1,111 @@
+import { Tabs } from "nextra/components";
+
+# Social Recovery Validator
+
+Social Recovery Validator allows users to recover their account using a social recovery mechanism
+
+You can find the source code for the module in the [Core Modules](https://github.com/rhinestonewtf/core-modules/tree/main/src/SocialRecovery) and the deployment address in the [address book](../../overview/address-book.mdx).
```
Comment (kopy-kat) [inline pages/module-sdk/modules/social-recovery-validator.mdx]:
```suggestion
You can find the source code for the module in the [Core Modules repo](https://github.com/rhinestonewtf/core-modules/tree/main/src/SocialRecovery) and the deployment address in the [address book](../../overview/address-book.mdx).
```

File: pages/module-sdk/modules/social-recovery-validator.mdx [hunk 2]
```diff
@@ -0,0 +1,111 @@
+import { Tabs } from "nextra/components";
+
+# Social Recovery Validator
+
+Social Recovery Validator allows users to recover their account using a social recovery mechanism
```
Comment (kopy-kat) [inline pages/module-sdk/modules/social-recovery-validator.mdx]:
```suggestion
The Social Recovery Validator allows users to recover their account using a social recovery mechanism
```

Comment (kopy-kat) [inline pages/module-sdk/modules/social-recovery-validator.mdx]:
```suggestion
Social Recovery Validator allows users to recover their account using a social recovery mechanism.
```

File: pages/module-sdk/modules/social-recovery-validator.mdx [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+You can find the source code for the module in the [Core Modules](https://github.com/rhinestonewtf/core-modules/tree/main/src/SocialRecovery) and the deployment address in the [address book](../../overview/address-book.mdx).
+
+## `getInstallSocialRecoveryValidator`
+
+The `getInstallSocialRecoveryValidator` function is used to install social recovery validator in the account. This function returns a module object.
```
Comment (kopy-kat) [inline pages/module-sdk/modules/social-recovery-validator.mdx]:
```suggestion
The `getInstallSocialRecoveryValidator` function is used to install the social recovery validator in the account. This function returns a module object.
```

File: pages/module-sdk/modules/social-recovery-validator.mdx [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+```typescript
+import { getInstallSocialRecoveryValidator } from "@rhinestone/module-sdk";
+
+const guardians = ["0x...", "0x..."]; // guardians address list
+const threshold = 2;
+
+const socialRecoveryValidator = getInstallSocialRecoveryValidator({
+  guardians,
+  threshold,
+  hook,
```
Comment (kopy-kat) [inline pages/module-sdk/modules/social-recovery-validator.mdx]: this is only for kernel right

Comment (YasseinBilal) [reply pages/module-sdk/modules/social-recovery-validator.mdx]: @kopy-kat Yes correct

**Pull request merged**